tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Steps/OrderSteps.cs

```diff
@@ -51,7 +51,7 @@ public async Task GivenOrdersExit(Table table)
         [When(@"a GET request is made for a list of orders with organisationId (.*)")]
         public async Task WhenAGETRequestIsMadeForOrders(Guid organisationId)
         {
-            await _request.GetAsync($"{_orderUrl}/{organisationId}/orders");
+            await _request.GetAsync(string.Format(_orderOrganisationsUrl, organisationId));
```

Comment:
Not a fan, but I can live with that 😄

Reply:
@assoomro it was @sam-roberts-bjss suggestion ;)